From 71da9339b62ed9b2c5b0e2252e0a3fbb24b6c80a Mon Sep 17 00:00:00 2001
From: Ping Gao <ping.a.gao@intel.com>
Date: Thu, 2 Jul 2015 10:29:15 +0800
Subject: [PATCH 107/403] vgt: disable IPS

virtual power saving is not supported in the guest so IPS should be disabled.

Fix bug#687.

Signed-off-by: Ping Gao <ping.a.gao@intel.com>

Conflicts:
	drivers/gpu/drm/i915/i915_dma.c
---
 drivers/gpu/drm/i915/i915_dma.c |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/gpu/drm/i915/i915_dma.c b/drivers/gpu/drm/i915/i915_dma.c
index b813085..eb052bc 100644
--- a/drivers/gpu/drm/i915/i915_dma.c
+++ b/drivers/gpu/drm/i915/i915_dma.c
@@ -954,6 +954,9 @@ int i915_driver_load(struct drm_device *dev, unsigned long flags)
 
 	i915_check_vgpu(dev);
 
+	if (intel_vgpu_active(dev))
+		i915.enable_ips = 0;
+
 	ret = i915_gem_gtt_init(dev);
 	if (ret)
 		goto out_freecsr;
-- 
1.7.10.4

